src/profile/ostonya-profile.js

```diff
@@ -0,0 +1,53 @@
+async function getOstonya() {
+    const ostonyaApiUrl = "http://localhost:4000/students?slug=ostonya-thomas";
+    try {
+        const response = await fetch(ostonyaApiUrl, {
+            method: "GET"
+        });
+        const data = await response.json();
+        return data;
+    } catch (error) {
+        return console.error(error);
+    }
+}
+async function getNasaBackground() {
+  const nasaApiUrl = "https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY&date=2023-02-02";
+  try {
+      const response = await fetch(nasaApiUrl, {
+          method: "GET"
+      });
+      const data2 = await response.json();
+      return data2;
+  } catch (error) {
+      return console.error(error);
+  }
+}
+
+  document.addEventListener("DOMContentLoaded", async () => {
+    const students = await getOstonya();
+    const nasaImg = await getNasaBackground();
+    const ostonyaProfile = document.getElementById("ostonya-profile");
+    document.getElementById("card-outter").style.backgroundImage=`url('${nasaImg.url}')`;
+  console.log(nasaImg.url)
+  
+    let content = "";
+  students.forEach((student) => {
+    content += `
+    <div class = "card-outter" src= "../${nasaImg.url}">
```

Comment:
Don't need the `../`, that's only if you're referencing files withing the project. This path is coming from the server. Find out more by Googling something like "relative vs absolute paths javascript"
```suggestion
    <div class = "card-outter" src= "${nasaImg.url}">
```